enhancements/machine-api/aws-placement-groups.md

```diff
@@ -0,0 +1,487 @@
+---
+title: aws-placement-groups
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@sdodson"
+  - "@deads2k"
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - "@deads2k"
+creation-date: 2022-01-05
+last-updated: 2022-01-05
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+---
+
+# AWS Placement Groups
+
+## Summary
+
+This enhancement describes the process of integrating [AWS Placement Groups](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html#placement-groups-partition)
+into the OpenShift Machine API.
+
+Placement Groups allow users to define how groups of Machines should be placed within the Availability Zone.
+For example, they can clustered together within the same network spine, or separated for redundancy.
+
+## Motivation
+
+By adding support for Placement Groups within Machine API, end users will have finer control over the topology of their
+machines and will be able to leverage the benefits that different placement group strategies bring.
+
+### Goals
+
+- Allow users of Machine API to create a MachineSet with either a Cluster, Partition or Spread Placement Group
+- Allow usage of existing Placement Groups across 1 or more MachineSet
+  - To allow using different instance types within a placement group
+  - To allow using the same placement group across multiple availability zones
+  - To allow control of machines/workloads across partitions of a Partition placement group
+- Create Placement Groups based on user configuration when they do not exist
+- Remove Placement Groups when they are no longer required
+- Ensure Placement Groups can be removed during the cluster deprovisioning process
+- Allow appropriate usage of Partition Placement Groups across multiple MachineSets.
+
+### Non-Goals
+
+- Allowing this feature to be leveraged during installation (day 0)
+  - Users will be able to modify MachineSets after manifest creation if required but this will not be exposed as a
+    part of the installation configuration before manifest generation
+- Support for Placement Groups for Control Plane Machines
+  - Users may replace their control plane at a later date using future automation if they wish to have their control
```

Comment:
If you claim this, I'd like like to see exactly how this will happen.  If you choose to permanently exclude it, you don't have to design it.